compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1732,16 +1733,15 @@ rustc_queries! {
     /// Does lifetime resolution on items. Importantly, we can't resolve
     /// lifetimes directly on things like trait methods, because of trait params.
     /// See `rustc_resolve::late::lifetimes` for details.
-    query resolve_bound_vars(_: hir::OwnerId) -> &'tcx ResolveBoundVars {
+    query resolve_bound_vars(def_id: hir::OwnerId) -> &'tcx ResolveBoundVars {
```

Comment:
```suggestion
    query resolve_bound_vars(owner_id: hir::OwnerId) -> &'tcx ResolveBoundVars {
```
Naming mismatch here an in a few other places below.